apps/chat/src/components/Common/ApplicationWizard/form.ts

```diff
@@ -369,8 +369,12 @@ export const getDefaultValues = ({
 export const getApplicationData = (
   formData: FormData,
   type: ApplicationType,
-): Omit<CustomApplicationModel, 'id' | 'reference'> => {
-  const preparedData: Omit<CustomApplicationModel, 'id' | 'reference'> = {
+): Omit<CustomApplicationModel, 'id' | 'reference' | 'owner'> => {
+  //TODO implement owner=author if needed
+  const preparedData: Omit<
+    CustomApplicationModel,
+    'id' | 'reference' | 'owner'
+  > = {
```

Comment:
```suggestion
): Omit<CustomApplicationModel, 'id' | 'reference'> => {
  //TODO implement owner=author if needed
  const preparedData: Omit<
    CustomApplicationModel,
    'id' | 'reference'
  > = {
```

guess this is optional field, so we haven't Omit it

Reply:
At the begginig it was not optional. My bad. Removed.